flags/flags.go

```diff
@@ -207,6 +207,11 @@ be stored in the 'sshpop' header.`,
 		Name:  "team",
 		Usage: "The team <name> used to bootstrap the environment.",
 	}
+
+	RedirectURL = cli.StringFlag{
+		Name:  "redirect-url",
+		Usage: "Terminal OAuth <redirect-url>.",
```

Comment:
It will look nicer if the usage was `"Terminal OAuth redirect <url>."` so the flag is displayed as `--redirect-url=url`.

Reply:
done